SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLSetGlobalPropertiesSpec.m

```diff
@@ -89,6 +93,28 @@
         expect(testRequest.keyboardProperties).to(beNil());
         expect(testRequest.userLocation).to(beNil());
     });
+
+    it(@"Should return new init", ^ {
+        SDLSetGlobalProperties* testRequest = [[SDLSetGlobalProperties alloc] initWithHelpText:helpTest timeoutText:timeoutTest vrHelpTitle:vrHelpTitle vrHelp:[@[help] mutableCopy] menuTitle:menuTitle menuIcon:image keyboardProperties:keyboard menuLayout:nil];
+        
+        SDLSetGlobalProperties* testRequest2 = [[SDLSetGlobalProperties alloc] initWithHelpText:helpTest timeoutText:timeoutTest vrHelpTitle:vrHelpTitle vrHelp:[@[help] mutableCopy] menuTitle:menuTitle menuIcon:image keyboardProperties:keyboard userLocation:nil menuLayout:nil];
+
+        expect(testRequest).to(equal(testRequest2));
+    });
+    
+    it(@"Should set initWithHelpText", ^ {
```

Comment:
If there are a lot of inits, use the full method name so it is easier to see which method is being tested. Something like:

"Should initialize correctly with initWithHelpText:timeoutText:vrHelpTitle:vrHelp:menuTitle:menuIcon:keyboardProperties:userLocation:menuLayout:"